src/styles/main.scss

```diff
@@ -0,0 +1,51 @@
+@import "./utils/_variables";
+@import "./utils/_mixins";
+@import "./utils/_animation";
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.watch {
+  position: relative;
+  @include circle($watch-size);
+  background: #eee;
+  border: 10px dotted green;
+
+  &__center {
+    position: absolute;
+    top: calc(50% - #{$center-width} / 2);
+    left: calc(50% - #{$center-width} / 2);
+    @include circle($center-width);
+    background-color: #5f98a8;
```

Comment:
You use this color in several places, so create a variable for this color, as well